Library/Homebrew/cask/auditor.rb

```diff
@@ -1,16 +1,20 @@
 require "cask/download"
+require "net/http"
 
 module Cask
+  # rubocop:disable Metrics/ParameterLists
   class Auditor
-    def self.audit(cask, audit_download: false, check_token_conflicts: false, quarantine: true, commit_range: nil)
-      new(cask, audit_download, check_token_conflicts, quarantine, commit_range).audit
+    def self.audit(cask, audit_download: false, audit_appcast: false,
```

Comment:
Just FTR, this signature is becoming quite bloated.

Reply:
any ideas how to avoid that without significantly complicating the implementation?

Reply:
Convert the `initialize` below to use named parameters.

Reply:
Convert the `initialize` below to use named parameters.

Reply:
thanks i'll ready up on ruby and try to do that.